src/main/java/com/openlattice/postgres/PostgresDatatype.java

```diff
@@ -123,6 +125,8 @@ public static PostgresDatatype getEnum(String dataTypeInput ) {
                 return TIMESTAMPTZ;
             case "TIMESTAMP WITHOUT TIME ZONE":
                 return TIMESTAMP;
+            case "USER-DEFINED":
+                return USER_DEFINED;
```

Comment:
What about `REAL`?

Reply:
REAL is just REAL

Reply:
Yeah, this function just maps strings that need to be transformed to Java enum names. `USER-DEFINED` is an invalid enum name